xml/ha_maintenance.xml

```diff
@@ -77,7 +121,7 @@
 <screen>&prompt.root;crm -w node standby</screen>
     <para>
      That way, services can migrate off the node without being limited by the
-     shutdown timeout of &pace;.
+     shutdown timeout of cluster services (&pace;).
```

Comment:
```suggestion
     shutdown timeout of &pace; cluster services.
```

Reply:
This is my compromise suggestion between old & new.

I think the word pacemaker, dangling on its own in parentheses on the end of the line, looks odd and does not contribute to the explanation.